pkg/web/api/v1/prepared.go

```diff
@@ -20,6 +20,7 @@ type Prepared struct {
 	TravelAddress string                   `json:"travel_address"`
 	Identity      *ivms101.IdentityPayload `json:"identity"`
 	Transaction   *generic.Transaction     `json:"transaction"`
+	Dump          string                   `json:"dump"`
```

Comment:
You can call methods in Go templates, so you don't have to indirect it like this. I'll update this PR with that method so you can see it since I don't know if I can explain it exactly in the PR review.